src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraphCheck.java

```diff
@@ -1,24 +1,36 @@
 package com.google.checkstyle.test.chapter7javadoc.rule712paragraphs;
 
+// 2 violations 5 lines below:
+//                           '.* tag should be placed immediately before the first word'
```

Comment:
Can we use exact value instead of .* ?

Let's treat .* as completely forbidden characters at the beginning of end of message

Reply:
I guess we cannot. In this case `<p>`  is the tag we're talking about, but the `<` and `>` causes mismatching in comparing violation messages as we compare messages using regex. 

I have used `.*` in my other PRs too because of the similar reason. Special symbols in violation messages causes mismatching in comparing messages due to regex used in comparing messages.

Reply:
At PRs we merged it was clear what we violating, now it is not clear, even I don't understand it.
Can we we use `\` to escape them? It is better than not clear what is causing violation.

Reply:
done, escaped `<p>` using `\` as you suggested:

https://github.com/checkstyle/checkstyle/blob/b0464ec57c620df634808ed56b071faf17564c35/src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraphCheck.java#L3-L5